Source/Krypton Components/Directory.Build.targets

```diff
@@ -6,8 +6,8 @@
     <PropertyGroup>
         <PackageId Condition="'$(TFMs)' == 'lite'">$(PackageId).Canary.Lite</PackageId>
         <Description Condition="'$(TFMs)' == 'lite'">
-          An update to Component factory's krypton toolkit to support .NET Core 3.1, .NET 5/6 and the latest .NET 3.5/4.x framework. $(Description)
-          
+          An update to Component factory's krypton toolkit to support .NET Core 3.1, .NET 5, .NET 6, and .net 4.8 Framework. $(Description)
+
           This package supports only .NET Framework >= 4.8, .NET Core 3.1 and .NET 5/6.
           If you require .NET 4.0 support or want to use library fully build against your specific framework version use non-lite package.
```

Comment:
This should read
"If you require .NET < 4.8 support...."

Reply:
or 
"If you require .NET 3.5 -> 4.7 support ....."

Reply:
Had to use other wording as VS doesn't let you use `<` because it's a XML document.